devito/operations/interpolators.py

```diff
@@ -207,8 +261,18 @@ def _rdim(self):
 
     def _augment_implicit_dims(self, implicit_dims, extras=None):
         if extras is not None:
+            # Get grid dimensions from the variables supplied
+            edims = []
+            for v in extras:
+                try:
+                    edims += [d for d in v.grid.dimensions]
+                except AttributeError:
+                    # Not on a grid, grab its space dimensions instead
```

Comment:
"its Dimensions", not "its space dimensions" (for all we know they might not be SpaceDimensions)